### ISSUE TITLE
[Feature request] Use Limited Egg Incubators for [10/5/2]km / Use Permanent Egg Incubator for [10/5/2]km

### ISSUE THREAD

**Issue by ingokotin** [NoxxDev/NecroBot#752]
Hey there, would be great if you could add this feature, so that we'd choose what eggs are going into which incubator. This option could save us a lot of incubators.

**Comment by Brazuca87** [comment]
There used to be a feature where you would select the minimum km for Egg Incubators, I think that was removed a while back.

Currently, it will prioritize:
Perma: 2 > 5 > 10
Limited: 10 > 5 > 2 (actually not even sure if 2km eggs will be used by limited use ones)

**Comment by Brazuca87** [comment]
@ingokotin not sure if the option still works, as it was removed from my config, but you can give it a try:
```"UseEggIncubatorMinKm": 5,```
This used to make so that my Limited Use Incubators would only use 5km eggs and above.

**Comment by ingokotin** [comment]
Will give it a try, but I'm afraid that the 2km Eggs won't be used at all, what means as soon as I only have 2km Eggs it will stop breeding.

**Comment by ingokotin** [comment]
@Brazuca87 I will give it a try, but I'm afraid that the 2km Eggs won't be used at all then, what means as soon as I only have 2km Eggs left it will stop breeding.
No error so far after adding `"UseEggIncubatorMinKm": 5,`.

**Comment by mo0ojava** [comment]
That switch is removed entirely. The reason was that a hatchlings level is determined by your lvl when you pick it up. Thus no point in differentiating. I wanted to save the Limited incubators to lvl 20 and that commit was accepted. So now you can switch using them on whenever you want.

**Comment by ingokotin** [comment]
@mo0ojava For me actually there is a point, I'd like to use the permanent one for 2km and 5km ones, the 10km ones in the limited ones only. Adding this feature would not exclude your commit.

I checked the wiki.

**Comment by Brazuca87** [comment]
@ingokotin If you set he way you said, that could eventually cause a "happy problem". If you get filled with 10km eggs, and no limited use incubators, you would never hatch an egg :)

**Comment by ingokotin** [comment]
@Brazuca87 Can't imagine ever only have 10km eggs :P
And even if that'd ever happen, I'd just change the option ;)

**Comment by CreativiTimothy** [comment]
For some reason, the minkm feature was removed. It needs to be put back in. Like @ingokotin I want to use the permanent one for 2km and 5km, with the 10km with limited incubators.

    "UseEggIncubators": true,
    "UseLimitedEggIncubators": true,
    "UseEggIncubatorMinKm": 10,

Putting "UseEggIncubatorMinKm" doesn't give me any errors, but I'm not sure if it's actually working. Hope someone can check the code and re-add it again

**Comment by mo0ojava** [comment]
Here's where the feature you're mentioning got removed:  6a9dcb1
Set that to 10 = only use limited on 10km Eggs.

**Comment by VBQL** [comment]
I don't think using different kind of incubators will affect the speed which the eggs will hatch at, you have to clear all of the 2KM eggs out anyways.

**Comment by ingokotin** [comment]
@VBQL That's true but speed is not my point. It's just about saving the limited ones for the 10km eggs and not wasting them for 2km or 5km ones.

**Comment by VBQL** [comment]
@ingokotin Ok yeah I got your point now. But I do have a question, why do you care about saving the incubators when they could be effective?

**Comment by Brazuca87** [comment]
To avoid "wasting" uses on lower eggs would be my guess... Which kinda makes sense


Get Outlook for Android






On Fri, Sep 2, 2016 at 10:34 AM -0400, "Viewerisland" <notifications@github.com> wrote:





@ingokotin Ok yeah I got your point now. But I do have a question, why do you care about saving the incubators when they could be effective?
